docker-compose/dependent-docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
 
 
   esignet:
-    image: 'mosipdev/esignet:release-1.5.x'
+    image: 'mosipdev/esignet-with-plugins'
```

Comment:
Lets decide and add tag to the docker image.

Reply:
updated